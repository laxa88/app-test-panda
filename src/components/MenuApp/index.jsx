import React from 'react';

import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

const MenuApp = () => (
  <div>
    <FontAwesomeIcon icon={['fas', 'bars']} />
    Dummy App
  </div>
);

export default MenuApp;
