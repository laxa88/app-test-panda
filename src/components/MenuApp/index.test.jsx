import React from 'react';
import renderer from 'react-test-renderer';

import MenuApp from './index';

describe('<MenuApp />', () => {
  it('renders', () => {
    const result = renderer.create(<MenuApp />);
    expect(result.toJSON()).toMatchSnapshot();
  });
});
