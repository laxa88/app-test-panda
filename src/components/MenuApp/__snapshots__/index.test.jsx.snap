// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuApp /> renders 1`] = `
<div>
  Mocked_FontAwesomeIcon (fas,bars)
  Dummy App
</div>
`;
